app/test/pages/index.test.tsx

```diff
@@ -1,23 +1,23 @@
 // test/pages/index.test.js
 
-import { render, screen } from "@testing-library/react";
-import { axe } from "jest-axe";
-import Home from "@pages/index";
+import { render, screen } from '@testing-library/react'
+import { axe } from 'jest-axe'
+import Home from '@pages/index'
```

Comment:
Probably my lack of typescript background kicking in.....but
would eslint removing the semicolons "break" anything? I thought it was required for statements such as the one found in lines 7-23

Reply:
@aplybeah it shouldn't break anything -- the tests still run as expected :)